flang/include/flang/Runtime/command.h

```diff
@@ -23,6 +24,10 @@ extern "C" {
 // integer kind.
 std::int32_t RTNAME(ArgumentCount)();
 
+// ??.?.??
```

Comment:
does anyone know that this numbers means?

Reply:
That number is generally from the standard, the section number. Since this is not part of the Fortran standard, there won't be a number here.

You might also want to add a comment on why this could not be implemented like `iargc`.

Reply:
I have tried to implement it in `flang/runtime/extension`
```c++
// RESULT = GETPID()
pid_t FORTRAN_PROCEDURE_NAME(getpid)() { return RTNAME(GetPID)(); }
```
However, when testing getpid in a file, getpid would return a `float` rather than an `i32`, the same applies even if I change the return type from `pid_t` -> `std::int32`.
There is a workaround, by renaming the function from `getpid` -> `igetpid`
|...| Before  |  After |
|---|---|---|
| fuc name  | getpid  | igetpid  |  
| -S -emit-llvm  | ` %3 = call contract float @getpid_()`  | `%3 = call i32 @iargc_()` |